### PR TITLE
pod-security-admission: clarify final restricted profile goal

### PR DESCRIPTION
**Pull request by s-urbaniak**
The final clarification of enabling restricted pod security profile by default was not clear enough, this fixes it.

/cc @bparees @deads2k @stlaz @mfojtik

### CODE REVIEW

File: enhancements/authentication/pod-security-admission.md [hunk 1]
```diff
@@ -111,6 +117,18 @@ spec:
               warn-version: latest
 ```
 
+4. Increase enforce audit profile to "restricted"
```
Comment (bparees) [inline enhancements/authentication/pod-security-admission.md:120]: did you mean just `enforce` here?  `audit` became restricted in an earlier step.

**Comment by deads2k** [comment]
The overall flow seems reasonable, needs comments addressed before lgtm.

/approve

File: enhancements/authentication/pod-security-admission.md (hunk 1 above)
Comment (s-urbaniak) [reply enhancements/authentication/pod-security-admission.md:120]: yes, just `enforce`. As @stlaz mentioned in https://github.com/openshift/enhancements/pull/1132#pullrequestreview-981843217 `warn` and `audit` are really not necessary in this case.

File: enhancements/authentication/pod-security-admission.md [hunk 2]
```diff
@@ -111,6 +117,18 @@ spec:
...
+
+Finally, after iterating over and adjusting workloads the "restricted" pod security profile is going to be enabled with the following settings:
+
+| Policy  | Profile |
+| --------| --------|
+| enforce | restricted |
+| warn    | restricted |
+| audit   | restricted |
```
[the remark this answers is not in the document]
Comment (s-urbaniak) [reply enhancements/authentication/pod-security-admission.md]: fixed

File: enhancements/authentication/pod-security-admission.md (hunk 1 above)
Comment (s-urbaniak) [reply enhancements/authentication/pod-security-admission.md:120]: reasoning: step 3. (the step we're in right now in the cycle) is a "grace period" where we can introspect out-of-process audit logs (aka `audit`) and in-process client side warnings (aka `warn`) to have a signal what workload needs to be converted. Once we switch to `enforce` admission will hard-reject non-compliant workloads, never reaching `audit` or `warn`.

Comment (stlaz) [reply enhancements/authentication/pod-security-admission.md:120]: I believe s/enforce audit profile/enforce profile/ is the suggested change which I would also like to see

**Review by stlaz — CHANGES_REQUESTED** [review]

File: enhancements/authentication/pod-security-admission.md (hunk 1 above)
Comment (s-urbaniak) [reply enhancements/authentication/pod-security-admission.md]: fixed

**Pull request merged**